client/src/modules/list/components/ItemsContainer.jsx

```diff
@@ -85,6 +87,11 @@ class ItemsContainer extends Component {
                 id: 'list.list-item.input-find-items'
               })}
             />
+            <Chips
+              count={itemsCount}
+              icon={<ListItemsIcon />}
+              title={formatMessage({ id: 'list.items-counter-label' })}
```

Comment:
Please parameterize the title, for example: "Showing {current} of {total} items".